content/docs/reference/katib/v1alpha3/katib.md

```diff
@@ -0,0 +1,717 @@
+  - [GetObservationLogRequest](#getobservationlogrequest)
+  - [GetSuggestionsReply](#getsuggestionsreply)
+  - [GetSuggestionsReply.ParameterAssignments](#getsuggestionsreplyparameterassignments)
+  - [GetSuggestionsRequest](#getsuggestionsrequest)
+  - [GraphConfig](#graphconfig)
+  - [Metric](#metric)
+  - [MetricLog](#metriclog)
+  - [NasConfig](#nasconfig)
+  - [NasConfig.Operations](#nasconfigoperations)
+  - [ObjectiveSpec](#objectivespec)
+  - [Observation](#observation)
+  - [ObservationLog](#observationlog)
+  - [Operation](#operation)
+  - [Operation.ParameterSpecs](#operationparameterspecs)
+  - [ParameterAssignment](#parameterassignment)
+  - [ParameterSpec](#parameterspec)
+  - [ReportObservationLogReply](#reportobservationlogreply)
+  - [ReportObservationLogRequest](#reportobservationlogrequest)
+  - [Trial](#trial)
+  - [TrialSpec](#trialspec)
+  - [TrialSpec.ParameterAssignments](#trialspecparameterassignments)
+  - [TrialStatus](#trialstatus)
+  - [ValidateAlgorithmSettingsReply](#validatealgorithmsettingsreply)
+  - [ValidateAlgorithmSettingsRequest](#validatealgorithmsettingsrequest)
+  - [ObjectiveType](#objectivetype)
+  - [ParameterType](#parametertype)
+  - [TrialStatus.TrialConditionType](#trialstatustrialconditiontype)
+  - [EarlyStopping](#earlystopping)
+  - [Manager](#manager)
+  - [Suggestion](#suggestion)
+- [Scalar Value Types](#scalar-value-types)
+  
+    - [ObjectiveType](#api.v1.alpha3.ObjectiveType)
+    - [ParameterType](#api.v1.alpha3.ParameterType)
+    - [TrialStatus.TrialConditionType](#api.v1.alpha3.TrialStatus.TrialConditionType)
+  
+  
+    - [EarlyStopping](#api.v1.alpha3.EarlyStopping)
+    - [Manager](#api.v1.alpha3.Manager)
+    - [Suggestion](#api.v1.alpha3.Suggestion)
+  
+
+- [Scalar Value Types](#scalar-value-types)
+
+
+
+<a name="api.proto"></a>
+<p align="right"><a href="#top">Top</a></p>
+
+## api.proto
+Katib API
+
+
+<a name="api.v1.alpha3.AlgorithmSetting"></a>
+
+### AlgorithmSetting
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| name | [string](#string) |  |  |
+| value | [string](#string) |  |  |
+
+
+
+
+
+
+<a name="api.v1.alpha3.AlgorithmSpec"></a>
+
+### AlgorithmSpec
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| algorithm_name | [string](#string) |  |  |
+| algorithm_setting | [AlgorithmSetting](#api.v1.alpha3.AlgorithmSetting) | repeated |  |
+| early_stopping_spec | [EarlyStoppingSpec](#api.v1.alpha3.EarlyStoppingSpec) |  |  |
+
+
+
+
+
+
+<a name="api.v1.alpha3.DeleteObservationLogReply"></a>
+
+### DeleteObservationLogReply
+
+
+
+
+
+
+
+<a name="api.v1.alpha3.DeleteObservationLogRequest"></a>
+
+### DeleteObservationLogRequest
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| trial_name | [string](#string) |  |  |
+
+
+
+
+
+
+<a name="api.v1.alpha3.EarlyStoppingSpec"></a>
+
+### EarlyStoppingSpec
+TODO
```

Comment:
In answer to your question about the TODOs (this one and the second one on this page): I think it would be nice to remove the TODOs from the code source file, so that they don't appear in the generated docs. After all, there's a lot to do still. :)

If the "TODO" means that this feature is only partially implemented, then we should add words in the code file that explain that, rather than just a vague "TODO" which doesn't tell people what's still to do.

Reply:
I've submitted a PR with the suggested change. Let's see if it gets merged:
https://github.com/kubeflow/katib/pull/1017

Reply:
Thanks @sarahmaddox for submitting a PR for that. I will "resolve conversation" for this PR for now as I believe the TODO issues will be addressed by kubeflow/katib#1017

Please "unresolve" the conversation if this is not the correct way to handle this.